.stylelintrc

```diff
@@ -1,3 +1,20 @@
 {
-  "extends": "stylelint-config-standard"
+  "extends": "stylelint-config-standard",
+  "rules": {
+    "at-rule-empty-line-before": [
```

Comment:
Altså, hvis du ser på reglene som finnes i stylelint-config-standard, så finner du igjen at-rule-empty-line-before, og den er helt lik utenom linjen som heter "ignoreAtRules". Her har jeg lagt til "include", fordi  jeg ikke vil at den skal feile på linjer som begynner på "@include".

Regelen slik den opprinnelig stod, så ville den at det "always" skulle være en blank linje før en at-rule ("@import", "@include", "@mixin", etc). Men "exception" reverserer logikken, så når det står "except" "blockless-group" så betyr det (så vidt jeg har skjønt) at det IKKE kan være en tom linje over en at-rule dersom de er like, altså to eller flere "@import" etter hverandre, f.eks.

Problemet var bare at den så ikke ut til å like at det var linje mellom "@import" og "@include" i _type.scss. Skjønner ikke hvorfor. Derfor la jeg til "@include" i ignoreAtRules-listen.

Mulig det er en bedre måte å løse det på?

Mer info finnes her:
https://github.com/stylelint/stylelint/tree/master/src/rules/at-rule-empty-line-before